web/src/components/storage/ProposalTransactionalInfo.jsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+import { sprintf } from "sprintf-js";
+import { _ } from "~/i18n";
+import { Reminder } from "~/components/core";
+import { isTransactionalSystem } from "~/components/storage/utils";
+import { useProduct } from "~/context/product";
+
+/**
+ * @typedef {import ("~/client/storage").ProposalManager.ProposalSettings} ProposalSettings
+ */
+
+/**
+ * Information about the system being transactional, if needed
+ * @component
+ *
+ * @param {object} props
+ * @param {ProposalSettings} props.settings - Settings used for calculating a proposal.
+ */
+export default function ProposalTransactionalInfo({ settings }) {
```

Comment:
Not sure if this should from now on live at _src/components/product/Transactional{Info|Reminder}.jsx_, but it is something that I'd evaluate, if needed, in the context of https://trello.com/c/ynFUHahV/353-agama-better-transactional-explanations-at-storage.

Looks related to the product itself... but actually it checks the storage settings.